### PR TITLE
community section is hidden

### PR DESCRIPTION
**Pull request by balaharish7**

### CODE REVIEW

File: website/src/pages/Community/index.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import {
-  Button, Grid, Link, Paper, Typography,
+  Button, Grid, Paper, Typography,
```
Comment (avishnu) [inline website/src/pages/Community/index.tsx:2]: Why is this change needed?

Comment (balaharish7) [reply website/src/pages/Community/index.tsx:2]: This change is needed, because the deploy and the build got failed. In the deploy log, it was suggested to remove this link as this link is related to community events.

**Review by avishnu — APPROVED** [review]
lgtm
